src/components/Community/AddPost/PostInfo.tsx

```diff
@@ -0,0 +1,115 @@
+// import { db, collection, addDoc } from './firebaseConfig';
+import { useState } from 'react';
+import PostInfoSection from './PostInfoSection';
+import PostPickWeather from './PostPickWeather';
+import PostPickDate from './PostPickDate';
+import PostPickMatch from './PostPickMatch';
+import PostMatchReview from './PostMatchReview';
+
+const PostInfo = () => {
+  const [title, setTitle] = useState('');
+  const [weather, setWeather] = useState<string | null>(null);
+  const [date, setDate] = useState<string>('');
+  const [match, setMatch] = useState<{ home: string; away: string }>({ home: '', away: '' });
+  const [winningTeam, setWinningTeam] = useState<string>('');
+  const [review, setReview] = useState<string>('');
+
+  const teams = ['kia', 'samsung', 'lg', 'doosan', 'ssg', 'kt', 'nc', 'hanhwa', 'lotte', 'kiwoom'];
+  const filteredTeams = teams.filter(team => team === match.home || team === match.away);
+
+  const handleSubmit = async (event: React.FormEvent) => {
+    event.preventDefault();
+
+    const formData = {
+      title,
+      weather,
+      date,
+      match,
+      winningTeam,
+      review,
+    };
+
+    try {
+      // const docRef = await addDoc(collection(db, 'posts'), formData);
+      // console.log('Document written with ID: ', docRef.id);
+    } catch (error) {
+      console.error('Error adding document: ', error);
+    }
+  };
+
+  const handleSelectClick = (event: React.MouseEvent<HTMLSelectElement>) => {
+    if (!match.home || !match.away) {
```

Comment:
개인적으로는 조건식을 아래와 같이 명시적으로 표현하는 것도 권장드리고 싶어요!
```suggestion
    if (match.home === '' || match.away === '') {
```

Reply:
JSX쪽을 보니까 select에 disabled 옵션이 동일한 조건으로 걸려있어서 이 함수가 실행될 케이스가 없을 것 같네요!